tembo-operator/src/cloudnativepg/hibernate.rs

```diff
@@ -63,7 +78,7 @@ pub async fn reconcile_cluster_hibernation(cdb: &CoreDB, ctx: &Arc<Context>) ->
     // Conversely, setting it back to 1 if the cluster is started should reverse
     // the process.
 
-    let replicas = if cdb.spec.stop { 0 } else { 1 };
+    let replicas = get_pooler_instances(cdb);
```

Comment:
I don't really like this. It's making an implicit assumption that poolers are the _only_ possible Deployment. Yes, the `get_pooler_instances` function does effectively the same thing, but the name of the function is inherently misleading.

Reply:
I dont understand.  The value in the spec for `Pooler` is `spec.instances`, so I am getting the number of instances it should be set to.  The function call does this.  I could also just do `if cdb.spec.stop { 0 } else { 1 }`, but I figured I would create a function since it needs to be figured out more than once.

Reply:
This section of code is for Deployments though, not pooler instances. Reusing that function here is misleading.

Reply:
Ah yeah, I understand what you're saying now.  Yes I updated the incorrect line.  This is reverted now.